pkg/settings/setting.go

```diff
@@ -448,3 +449,20 @@ func IterateWhitelistedEnvVars(handler func(name, value string)) {
 		}
 	}
 }
+
+// GetMachineProvisionImagePullPolicy will return the pull policy to be used on MachineProvisioning job.
+//
+//	If an invalid value is set it will return the default value v1.PullAlways
```

Comment:
This comment isn't formatted properly. We should remove the indentation and white space between the two sentences

Reply:
Done